src/components/PostDetails.tsx

```diff
@@ -1,96 +1,42 @@
+/* eslint-disable react-hooks/exhaustive-deps */
 import React, { useEffect, useState } from 'react';
 import { Loader } from './Loader';
 import { NewCommentForm } from './NewCommentForm';
 
-import * as commentsApi from '../api/comments';
-
 import { Post } from '../types/Post';
-import { Comment, CommentData } from '../types/Comment';
+import { useAppDispatch, useAppSelector } from '../app/hooks';
+import * as commentsActions from '../features/comments/commentsSlice';
+import * as commentsApi from '../api/comments';
 
 type Props = {
```

Comment:
use interface prefer. 
Types have unique features but the interface is best for scaling.